tools/run_command.py

```diff
@@ -0,0 +1,19 @@
+import logging
+import subprocess
+
+
+# Echoes and runs an OS command, returning exit status and the captured
+# stdout+stderr as a string array.
+def runCommand(command):
```

Comment:
In Python3, `subprocess.run` basically gives you a fair bit of this, since it doesn't check by default and can capture everything. Can you rewrite to base on this? It might turn out it's not that worthwhile to make a full standalone util here.